src/Automation/Resources/ErrorMessages.resx

```diff
@@ -121,22 +121,21 @@
     <value>The directory "{0}" was invalid.</value>
   </data>
   <data name="ErrorFailToGetRootElementsOfProcess" xml:space="preserve">
-    <value>ERROR Automation017: Failed to get the root element(s) of the specified process({0}) : {1}</value>
```

Comment:
AutomationIntegrationTests checks for this error in lines 248 and 264. Looking at the code, maybe we just always Assert.Inconclusive if an error is thrown while scanning ?